mlflow/server/js/src/experiment-tracking/components/experiment-page/utils/getCommonArtifacts.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Gets the list of artifacts that are present in all runs in the given list of runs.
+ * @param artifactsKeyedByRun Object containing artifacts keyed by run UUID
+ * @returns List of common artifacts
+ */
+export const getCommonArtifacts = (artifactsKeyedByRun: Record<string, any>) => {
```

Comment:
similar to above for typing

```suggestion
export const getCommonArtifacts = (artifactsKeyedByRun: Record<string, ArtifactListFilesResponse >) => {
```